src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsJavaDocsTest.java

```diff
@@ -415,8 +416,7 @@ private static String transformXmlToJavaDocName(String name) {
         return result;
     }
 
-    private static String getAttributeText(String nodeName, NamedNodeMap attributes,
-            boolean fixLinks) {
+    private static String getAttributeText(String nodeName, NamedNodeMap attributes) {
```

Comment:
I don't remember the original purpose of needing this boolean. I removed it and it caused no issues.